webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -56,7 +55,7 @@ def is_previous_step(cls, possible_next_step_name, stored_data):
         return False
 
     @classmethod
-    def number_of_users(cls, input_data):
+    def number_of_users(cls, input_data, *args, **kwargs):
```

Comment:
Could you maybe explain why you added `*args` and `**kwargs` in some occasions? I don't see what you changed about this method that requires that?

Reply:
Okay, this is a good occurrence because I made a mistake (we should get rid of input_data here). The idea basically was to get rid of any parameters in the implementation of the base class which are needed for the implementation in the children and instead use args and kwargs. I have also added it to the init of the steps when we were still passing form data. I think we could get rid of it now. However, I think it makes sense to keep it there to allow children to set more parameters.

That said I think that this is a point where we could think more about getting rid of the inheritance of the step.